### ISSUE TITLE
Continue to analyze and find root cause of garbage/orphan flows

### ISSUE THREAD

**Issue by viniarck** [kytos-ng/mef_eline#549]
This is to continue to analyze the garbage/orphan flows that @italovalcy has reported that has been found at AmLight after the 2023.2 upgrade. The issue of conflicting and missing vlans are is being addressed on this other PR https://github.com/kytos-ng/topology/pull/233. 

I'll only share part of the non confidential information here, basically, you (core developer who works at AmLight) need to investigate the log shared files and try to understand and narrow down how there unknown flows are install in installed:

```
Unknown: {"flow":{"owner":"mef_eline","cookie":12286695809644089670,"match":{"in_port":1,"dl_vlan":36},"actions":[{"action_type":"set_vlan","vlan_id":2},{"action_type":"output","port":2}],"table_id":0,"table_group":"evpl","priority":20000,"idle_timeout":0,"hard_timeout":0},"flow_id":"2793b922d27544f6be3b1f8d656bf46e","id":"96984ea6ea6acb3dab4f128c9c8f5749","inserted_at":"2024-10-21T02:38:22.395000","state":"installed","switch":"00:00:00:00:00:32:00:04","updated_at":"2024-10-21T03:56:23.538000"}

Unknown: {"flow":{"owner":"mef_eline","cookie":12286695809644089670,"match":{"in_port":1,"dl_vlan":9},"actions":[{"action_type":"set_vlan","vlan_id":54},{"action_type":"output","port":2}],"table_id":0,"table_group":"evpl","priority":20000,"idle_timeout":0,"hard_timeout":0},"flow_id":"eb66681baa188aa4f09845af96221051","id":"9b892a512e088f5f6ebee9ce7b21cdea","inserted_at":"2024-10-21T03:54:45.620000","state":"installed","switch":"00:00:00:00:00:32:00:04","updated_at":"2024-10-21T05:03:25.869000"}

Unknown: {"flow":{"owner":"mef_eline","cookie":12286695809644089670,"match":{"in_port":2,"dl_vlan":54},"actions":[{"action_type":"set_vlan","vlan_id":9},{"action_type":"output","port":1}],"table_id":0,"table_group":"evpl","priority":20000,"idle_timeout":0,"hard_timeout":0},"flow_id":"8bd3a595c795b3516a152c2f179ca82f","id":"a1d9c9b598d1464683530e939bf39c83","inserted_at":"2024-10-21T03:55:46.751000","state":"installed","switch":"00:00:00:00:00:32:00:04","updated_at":"2024-10-21T05:03:25.909000"}

Unknown: {"flow":{"owner":"mef_eline","cookie":12286695809644089670,"match":{"in_port":1,"dl_vlan":3},"actions":[{"action_type":"set_vlan","vlan_id":54},{"action_type":"output","port":2}],"table_id":0,"table_group":"evpl","priority":20000,"idle_timeout":0,"hard_timeout":0},"flow_id":"a79d0f17449e87c411d151d1d7e9710f","id":"0706f8f33ae6030fdb54e90fd634f7b9","inserted_at":"2024-10-21T03:56:47.700000","state":"installed","switch":"00:00:00:00:00:32:00:04","updated_at":"2024-10-21T05:03:25.950000"}

Unknown: {"flow":{"owner":"mef_eline","cookie":12286695809644089670,"match":{"in_port":1,"dl_vlan":7},"actions":[{"action_type":"set_vlan","vlan_id":2},{"action_type":"output","port":2}],"table_id":0,"table_group":"evpl","priority":20000,"idle_timeout":0,"hard_timeout":0},"flow_id":"f65aff43de0a1695f3dc3925726d0650","id":"90ff18918b90361325d8f7a194c0df75","inserted_at":"2024-10-21T03:49:41.090000","state":"installed","switch":"00:00:00:00:00:32:00:04","updated_at":"2024-10-21T05:05:25.970000"}

Unknown: {"flow":{"owner":"mef_eline","cookie":12286695809644089670,"match":{"in_port":2,"dl_vlan":2},"actions":[{"action_type":"set_vlan","vlan_id":9},{"action_type":"output","port":1}],"table_id":0,"table_group":"evpl","priority":20000,"idle_timeout":0,"hard_timeout":0},"flow_id":"67a1ceb932beecf221635183c008c6f3","id":"23cbfab600396bc100820d33e7fe91fc","inserted_at":"2024-10-21T02:38:22.395000","state":"installed","switch":"00:00:00:00:00:32:00:04","updated_at":"2024-10-21T05:08:25.876000"}
```

**Comment by viniarck** [comment]
Setting label `2024.2` to be on our radar. Plus, it might need to backport depending on how it unfolds. We'll see.

**Comment by viniarck** [comment]
After analyzing all the cases, I managed to find the root cause https://github.com/kytos-ng/flow_manager/issues/208, it reproduced the same issue, and from what followed in the logs they all are related to it. 

I'll fix it now.

**Comment by viniarck** [comment]
The related issue has been closed and PRs on `master` and backport `2023.2` landed. 

https://github.com/kytos-ng/flow_manager/issues/209
https://github.com/kytos-ng/flow_manager/pull/211

Only 2024.1 is still remaining https://github.com/kytos-ng/flow_manager/issues/210 (this will land soon too)
